app/src/main/res/layout/content_event.xml

```diff
@@ -361,7 +361,7 @@
                     android:id="@+id/imageMap"
                     android:layout_width="match_parent"
                     android:layout_height="@dimen/home_logo_width"
-                    app:hideIfEmpty="@{event.locationName}"
+                    android:visibility="visible"
```

Comment:
Why? It will be visible by default.

Reply:
actually it isn't when I checked the application 😇, there is already a check for locationName when the view is being created so it isn't really required...

Reply:
I have removed that statement too as, it is set visible by default 😅